python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
     """Return the number of vowels in string."""
-    vowels = "aeiou"
+    vowels = "aeiouy"
     num = 0
     for v in vowels:
         num += text.lower().count(v)        
     return num
 
 def num_consonants(text):
-    vowels = "aeiou"
+    vowels = "aeiouy"
```

Comment:
Please revert this change back to "aeiou" as vowels do not include "y".
```suggestion
    vowels = "aeiou"
```